Source/WebCore/html/track/WebVTTParser.h

```diff
@@ -111,6 +111,7 @@ class WebVTTParser final {
         Finished
     };
 
+    WebVTTParser() = delete;
```

Comment:
This seems unnecessary.

Reply:
True, it is not necessary for the core purpose of the patch—it was a drive-by fix.

I think we should delete constructors that no one should be using so they get a compiler error if they try to use it.